src/component/feature/FeatureToggleList/index.jsx

```diff
@@ -32,12 +32,19 @@ export const mapStateToPropsConfigurable = isFeature => state => {
             f => f.project === settings.currentProjectId
         );
     }
-
     if (settings.filter) {
         try {
             const regex = new RegExp(settings.filter, 'i');
-            features = features.filter(
-                feature =>
+            features = features.filter(feature => {
+                if (!isFeature) {
```

Comment:
huh, if not is feature?

Reply:
The isFeature property in the mapConfigurableProps determines whether or not it's we should treat it as a feature or archive. If it's not a feature, then it is an archived toggle and we should not check for strategies.